src/include/wiredtiger.in

```diff
@@ -7704,30 +7705,28 @@ typedef enum {
 /* VERBOSE ENUM START */
     WT_VERB_API,                  /*!< API messages. */
     WT_VERB_BACKUP,               /*!< Backup messages. */
-    WT_VERB_BLKCACHE,
-    WT_VERB_BLOCK,                /*!< Block manager messages. */
+    WT_VERB_BLOCK,			      /*!< Block manager messages. */
```

Comment:
I don't think we need to modify this file. We would like to keep the A->Z ordering.